RELEASE_NOTES.md

```diff
@@ -1,4 +1,6 @@
-#### 1.3.1.0 -
+#### 1.3.2 -
```

Comment:
I'm not really sure. I'm not fussed about following SemVer yet here - we haven't done that so far and no reason to start just yet I think.

I was thinking we'd use the "1" out the front for very major revisions and the fourth for very minor revisions, But perhaps we should just switch to only publish F# 4.0 support anyway at some point very soon.  In that case 1.3.1.x and 1.4.0.x seem to work well enough for now.